tests/helpers/test_template.py

```diff
@@ -841,7 +835,7 @@ def test_state_attr(hass):
             """,
         hass,
     )
-    assert tpl.async_render() == "True"
+    assert tpl.async_render() is True
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)